src/internal/components/live-region/index.tsx

```diff
@@ -1,176 +1,31 @@
- * differently and produce unexpected results. To imitate non-atomic announcements simply use
- * multiple live regions:
- * ```
- * <>
- *   <LiveRegion>{title}</LiveRegion>
- *   <LiveRegion><Details /></LiveRegion>
- * </>
- * ```
- *
- * If you place interactive content inside the LiveRegion, the content will still be
- * interactive (e.g. as a tab stop). Consider using the `source` property instead.
- */
-export default memo(LiveRegion);
-
-function LiveRegion({
-  assertive = false,
-  delay = 10,
-  visible = false,
-  tagName: TagName = 'span',
-  children,
-  id,
-  source,
-  ...restProps
-}: LiveRegionProps) {
-  const sourceRef = useRef<HTMLSpanElement & HTMLDivElement>(null);
-  const targetRef = useRef<HTMLSpanElement & HTMLDivElement>(null);
-
-  /*
-    When React state changes, React often produces too many DOM updates, causing NVDA to
-    issue many announcements for the same logical event (See https://github.com/nvaccess/nvda/issues/7996).
-
-    The code below imitates a debouncing, scheduling a callback every time new React state
-    update is detected. When a callback resolves, it copies content from a muted element
-    to the live region, which is recognized by screen readers as an update.
-
-    If the use case requires no announcement to be ignored, use delay = 0, but ensure it
-    does not impact the performance. If it does, prefer using a string as children prop.
-  */
-  useEffect(() => {
-    function getSourceContent() {
-      if (source) {
-        return source
-          .map(item => {
-            if (!item) {
-              return undefined;
-            }
-            if (typeof item === 'string') {
-              return item;
-            }
-            if (item.current) {
-              return extractInnerText(item.current);
-            }
-          })
-          .filter(Boolean)
-          .join(' ');
-      }
-
-      if (sourceRef.current) {
-        return extractInnerText(sourceRef.current);
-      }
-    }
-    function updateLiveRegion() {
-      const sourceContent = getSourceContent();
-
-      if (targetRef.current && sourceContent) {
-        const targetContent = extractInnerText(targetRef.current);
-        if (targetContent !== sourceContent) {
-          // The aria-atomic does not work properly in Voice Over, causing
-          // certain parts of the content to be ignored. To fix that,
-          // we assign the source text content as a single node.
-          targetRef.current.innerText = sourceContent;
-        }
-      }
-    }
-
-    let timeoutId: null | number;
-    if (delay) {
-      timeoutId = setTimeout(updateLiveRegion, delay);
-    } else {
-      updateLiveRegion();
-    }
-
-    return () => {
-      if (timeoutId) {
-        clearTimeout(timeoutId);
-      }
-    };
-  });
+function LiveRegion({ assertive = false, hidden = false, tagName = 'div', ...restProps }: LiveRegionProps) {
```

Comment:
Do we forward any html attributes to the underlying `div` or `span`, if so I don't think `LiveRegionProps` have the base html props type.